onnx/defs/reduction/defs.cc

```diff
@@ -6,6 +6,44 @@
 #include "onnx/defs/schema.h"
 
 namespace ONNX_NAMESPACE {
+void reduceShapeInference(InferenceContext& ctx) {
+  if (!hasNInputShapes(ctx, 1)) {
+    return;
+  }
+
+  int64_t keep_dims = 1;
```

Comment:
you should get the default value from the schema

Reply:
Good suggestion, however, I don't think the default value in opschema was designed to be accessible here. will discuss with @anderspapitto in separate thread and make it if doable in separate PR.